day_6/exercises/my_car.rb

```diff
@@ -0,0 +1,51 @@
+class MyCar
+  attr_accessor :color
+  attr_reader :year, :speed
+  def initialize(year, color, model, speed = 0)
+    @year = year
+    @color = color
+    @model = model
+    @speed = speed
+  end
+
+  def model
+    @model
+  end
+
+  def drive
+    "Currently at #{speed} mph."
+  end
+
+  def speed_up(mph)
+    @speed += mph
+  end
+
+  def brake(mph)
+    @speed -= mph
+  end
+
+  def shut_off_engine(mph)
+    @speed -= mph
+  end
+
+  def spray_paint(color)
+    self.color = color
+    puts "Your new Tesla paint job is #{self.color}."
+  end
+end
+
+tesla = MyCar.new(2019, "Burgundy", "Model S")
+
+puts tesla.drive
+tesla.speed_up(60)
+puts tesla.drive
+tesla.brake(25)
+puts tesla.drive
+tesla.shut_off_engine(35)
+puts tesla.drive
```

Comment:
Really like how you're feeling out when it makes sense for the method to have the `puts` and when you should puts the return value of the method. Since `puts` lines always return `nil`, it's usually more useful to have the method return a value so it can be reused in a different capacity.